src/pages/App.tsx

```diff
@@ -160,6 +162,10 @@ export default function App() {
     user.set(CustomUserProperties.DARK_MODE, isDarkMode)
   }, [isDarkMode])
 
+  useEffect(() => {
+    user.set(CustomUserProperties.ROUTER_PREFERENCE, routerPreference)
+  }, [routerPreference])
```

Comment:
`user` is outside of the React lifecycle - could this be set outside of a hook, in the reducer when the router preference is actually set? That would simplify and avoid coupling it to rendering.

Reply:
given the other user settings set here, I think it's fine to leave. This does set them all in the same file, so it's easier to track.